src/transformers/models/falcon/modeling_falcon.py

```diff
@@ -382,6 +406,7 @@ def forward(
         head_mask: Optional[torch.Tensor] = None,
         use_cache: bool = False,
         output_attentions: bool = False,
+        padding_mask: Optional[torch.LongTensor] = None,
```

Comment:
This is a big change in API design, but I'm not really against it.

Reply:
IMO ok as it's an internal class